### PR TITLE
VER-3304:Add Grafana views for each evidence source

### PR DESCRIPTION
**Pull request by YuanZHANG910**

### CODE REVIEW

File: it/controllers/QuestionControllerISpec.scala [hunk 1]
```diff
@@ -47,21 +45,6 @@ class QuestionControllerISpec extends BaseISpec with LogCapturing with BaseOneSe
       questions.map(q => q.questionKey) should contain(paymentToDateQuestion.questionKey)
     }
 
-//    "return 200 and a sequence of non p60 question if provided with valid json but P60 returns not found" in new Setup {
```
Comment (stevenmallen) [inline it/controllers/QuestionControllerISpec.scala:50]: Just curious - why is this test commented out?

File: app/uk/gov/hmrc/identityverificationquestions/monitoring/metric/MetricsService.scala [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import com.codahale.metrics.Timer
+import com.kenshoo.play.metrics.Metrics
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class MetricsService @Inject()(metrics: Metrics) {
+
+  lazy val payeConnectorTimer: Timer = metrics.defaultRegistry.timer("pay-as-you-earn-emp-connector-response-timer")
```
Comment (stevenmallen) [inline app/uk/gov/hmrc/identityverificationquestions/monitoring/metric/MetricsService.scala:28]: We may need to discuss this - not sure I understand why we are using timers here?  There are Meters, Counters, Derivatives, Ratios, etc available.  Why did you choose timer?  How will that work in Grafana to show us the 3 states?

File: it/controllers/QuestionControllerISpec.scala (hunk 1 above)
Comment (YuanZHANG910) [reply it/controllers/QuestionControllerISpec.scala:50]: I believe it was copied and pasted from IV

File: app/uk/gov/hmrc/identityverificationquestions/sources/sa/SAPaymentService.scala
Comment (stevenmallen) [inline app/uk/gov/hmrc/identityverificationquestions/sources/sa/SAPaymentService.scala:1]: Hi Yuan - I think you need to add the metrics for the gauge in here too, as it overrides the base implementation?

File: app/uk/gov/hmrc/identityverificationquestions/monitoring/metric/MetricsService.scala [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+object Good extends HealthState {
+  override def toString: String = "Good"
+}
+object Broken extends HealthState {
+  override def toString: String = "Broken"
+}
+object Unhealthy extends HealthState {
+  override def toString: String = "Unhealthy"
+}
+class HealthyGauge extends Gauge[HealthState] {
```
Comment (stevenmallen) [inline app/uk/gov/hmrc/identityverificationquestions/monitoring/metric/MetricsService.scala:64]: This looks perfect - thank you!

**Review by stevenmallen — APPROVED** [review]
LGTM

**Pull request merged**